Update the SDK Manager version

graugans opened the pull request:
The latest 0.9.14-4964 is used for installation. The definition
of the version was moved to a variable and one common variable for the .deb
file is used.

Signed-off-by: Christian Ege <christian.ege@ifm.com>

Commit:
Update the SDK Manager version

The latest 0.9.14-4964 is used for installation. The definition
of the version was moved to a variable and one common variable for the .deb
file is used.

Signed-off-by: Christian Ege <christian.ege@ifm.com>

deltaflyer commented:
Hello @graugans 

thank your for handing in this PR. I am also looking forward for the new SDK versions to get my Board Support Package updated on the Jetson Nano.
@atinfinity do you mind merging this PR?

Best,
Oliver

atinfinity commented on README.md:
```diff
@@ -10,7 +10,7 @@ This is a Dockerfile to use [NVIDIA SDK Manager](https://docs.nvidia.com/sdk-man
-This time, I used `sdkmanager_0.9.12-4180_amd64.deb`.
+This time, I used `sdkmanager_0.9.14-4964_amd64.deb`.
 
 ### Build Docker image
```
@graugans Thank you for your proposal! I think that your change is good.
I hope you add the procedure to specify `SDK_MANAGER_VERSION` as of `docker build` to README.md.

graugans replied:
@atinfinity sorry for the late response. I have not yet added advise how to set this from the docker command line. I will add this after cleaning my TODO list.

atinfinity replied:
@graugans 

> I have not yet added advise how to set this from the docker command line.

I tried to set `SDK_MANAGER_VERSION` using `--build-arg`.  
It works well.

```shell
$ docker build --build-arg SDK_MANAGER_VERSION=1.0.0-5517 -t jetpack .
```

graugans replied:
I've updated the PR

atinfinity commented:
@deltaflyer Thank you for your comment.

> do you mind merging this PR?

I agree this proposal.

Commit:
Add a passage with information about setting SDK_MANAGER_VERSION

This describes how to override the SDK_MANAGER_VERSION from the
command line of the Docker build

Signed-off-by: Christian Ege <christian.ege@ifm.com>
